submissions/aquaplanet_mountain.jl

```diff
@@ -0,0 +1,32 @@
+author = "Shirin Ermis"
+description = "Aqua-planet simulation with a mountain"
+
+using SpeedyWeather, RainMaker
+
+spectral_grid = SpectralGrid(trunc=31, nlayers=10)
+model = PrimitiveWetModel(spectral_grid)
+
+# Details for mountain
+H, λ₀, φ₀, σ = 4000, 2, 51, 5     # height, lon, lat position, and width
+
+# Set up aqauaplanet with large mountain in "North Sea"
+ocean = AquaPlanet(spectral_grid, temp_equator=302, temp_poles=300)
+orography = NoOrography(spectral_grid)
+# land_sea_mask = AquaPlanetMask(spectral_grid)
```

Comment:
While you define sea surface temperatures everywhere by defining the ocean as `AquaPlanet` you still have land because you don't pass on the land-sea mask to the model! In that case the sea surface temperatures are ignored where the land-sea mask says land!